gameinfo.conf.template

```diff
@@ -0,0 +1,7 @@
+NAME               GameName
+VERSION            0.0
+DEFAULT_BASEPAK    mybasepak
```

Comment:
why not just BASEPAK? We already know this is the default since it's stored in that file.

Reply:
After deep thoughts, this may be called `BASENAME`, there may be more than one usage for a basename.

We can also use it to compute `~/.local/share/${BASENAME}` while other OS may do something with `NAME` instead if it's more common on such OS to do that (`My Games/${GAME}`).

I don't think it's a bad idea to enforce the basename is also the basepak name.

It may also make easier to reimplement per-directory full-conversion downloadable mod like quake3, by using this BASENAME (`unvanquished` or `quake6`) string instead of a shortcut (`unv` or `baseq6`).